libs/ui-watt/src/lib/components/button/watt-button.component.scss

```diff
@@ -33,6 +33,33 @@ watt-button {
     }
   }
 
+  &.watt-button-loading {
+    .content-grid {
+      flex-direction: column;
+
+      > .content-grid {
+        flex-direction: row;
+      }
+
+      .content-grid-item:not(.content-grid-item-spinner) {
+        height: 0;
```

Comment:
Is this necessary?

Reply:
It is used for hiding the icon. Without it I get the following issue:

![loading](https://user-images.githubusercontent.com/1096332/140740820-99a4e0fc-3858-4a22-88af-392056d53b6d.gif)

Reply:
My mistake. Not setting the `height` results in the content being hidden visually but still taking space on the page.

![loading](https://user-images.githubusercontent.com/1096332/140741547-bee76ac5-fcb8-42ba-88a4-cf2a4d313d9f.gif)

Reply:
Wouldn't the "right" thing to do making the spinner absolute then? What if the button has multi-line text (not sure if that's posible), but then the height of the button will change?

Reply:
> Wouldn't the "right" thing to do making the spinner absolute then

Good idea. Hadn't thought of it. Will try it out.

> What if the button has multi-line text (not sure if that's possible), 

I'm not sure what will happen on multi-line text. Haven't tested it.

Reply:
Just tried with making the spinner absolute and it works better. The logic is also simpler. Thanks for the suggestion 👍